pkg/authorization/api/v1/conversion.go

```diff
@@ -130,7 +131,10 @@ func convert_api_ResourceAccessReviewResponse_To_v1_ResourceAccessReviewResponse
 	return nil
 }
 
-func convert_v1_PolicyRule_To_api_PolicyRule(in *PolicyRule, out *newer.PolicyRule, s conversion.Scope) error {
+func Convert_v1_PolicyRule_To_api_PolicyRule(in *PolicyRule, out *newer.PolicyRule, s conversion.Scope) error {
+	if defaulting, found := s.DefaultingInterface(reflect.TypeOf(*in)); found {
```

Comment:
If you're worried about reflection cost, why not go ahead and make a named function for it and make the call directly?  Even registering and calling an empty function would do a pretty good job of making sure the code doesn't slip.